lib/web_ui/lib/src/engine/canvaskit/color_filter.dart

```diff
@@ -83,10 +83,14 @@ class CkBlendModeColorFilter extends CkColorFilter {
 
   @override
   SkColorFilter _initRawColorFilter() {
-    return canvasKit.ColorFilter.MakeBlend(
+    SkColorFilter filter = canvasKit.ColorFilter.MakeBlend(
       toSharedSkColor1(color),
       toSkBlendMode(blendMode),
     );
+    if (filter == null) {
+      throw ArgumentError('Invalid parameters for blend mode ColorFilter');
```

Comment:
Do we know what combinations of color/blendMode we support? The message would be more helpful if we could give the user more information.

Reply:
The rules for which combinations are usable are implemented in https://github.com/google/skia/blob/main/src/core/SkModeColorFilter.cpp#L126

AFAIK Skia does not provide a concise summary of the valid arguments for this function.